english/ghana/end.md

```diff
@@ -0,0 +1 @@
+[Thank God. Am so happy to be at the end.] (https://github.com/udacity/create-your-own-adventure/blob/master/english/end/end.md)
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.